app_legacy/Helpers/render/game.php

```diff
@@ -190,6 +190,41 @@ function renderGameCard(int|array $game): string
     return $tooltip;
 }
 
+function RenderGameSort(bool $isFullyFeaturedGame, ?int $flags, int $officialFlag, int $gameID, ?int $sortBy): void
+{
+    echo "<div class='py-3'><span>";
+    echo "Sort: ";
+
+    $flagParam = ($flags != $officialFlag) ? "f=$flags" : '';
+
+    $sortType = ($sortBy < 10) ? "^" : "<sup>v</sup>";
+
+    $sort1 = ($sortBy == 1) ? 11 : 1;
+    $sort2 = ($sortBy == 2) ? 12 : 2;
+    $sort3 = ($sortBy == 3) ? 13 : 3;
+    $sort4 = ($sortBy == 4) ? 14 : 4;
+    $sort5 = ($sortBy == 5) ? 15 : 5;
+
+    $mark1 = ($sortBy % 10 == 1) ? "&nbsp;$sortType" : "";
+    $mark2 = ($sortBy % 10 == 2) ? "&nbsp;$sortType" : "";
+    $mark3 = ($sortBy % 10 == 3) ? "&nbsp;$sortType" : "";
+    $mark4 = ($sortBy % 10 == 4) ? "&nbsp;$sortType" : "";
+    $mark5 = ($sortBy % 10 == 5) ? "&nbsp;$sortType" : "";
+
+    echo "<a href='/game/$gameID?$flagParam&s=$sort1'>Normal$mark1</a> - ";
```

Comment:
In a Hubs context (and arguably an Achievements list context too), it is ambiguous and unclear what a "Normal" sort means.

Reply:
To help this new functionality be the best it can be, I propose we change the label for "Normal" while a user is in the context of a hub page. I am confused by the current link copy, as it does not provide a clear indication of the sorting criteria being applied to the list.

From my perspective, here is the current user journey:

- I land on a hub page and see a list sorted by points (primary) and game title (secondary).
- I click the "Normal" link. The page refreshes but nothing in the list changes.
- I click the "Normal" link again. The page refreshes, now the list inverts showing 0 to N points by Z-A game title.

To improve the user experience and mitigate confusion, I propose updating the label with a more descriptive name that reflects the sorting mechanism being applied.

Here are some possible alternatives I could think of:
* Points (A-Z)
* Points & Title
* Points, then Title
* Points, Alphabetical

Reply:
In case we are seeing two different things, here is how I repro'd the 2 click issue mentioned earlier. I am indeed finding I have to click it twice to make any change. This is all good, just sharing so we are on the same page. I don't think this issue is critical at all.

https://user-images.githubusercontent.com/3984985/227798909-1dfb7730-86f4-4dd2-ad22-8bfe01145f1e.mp4

Reply:
I don't have the ability to mark this comment as resolved, but it's now resolved from my POV.

Reply:
I just realized something about my local setup is actually evaluating differently from production for that, even on the achievements page it thinks normal is sorted already with no query parameters. I think it must be a slight evaluation difference with a null modulus or something because I am on php 8.2

Reply:
Updated the PR again though :)

Reply:
Heads up: we'll switch to PHP 8.2 in the upcoming weeks, to Laravel 10 after that.
I have to finish a larger task first.